package.json

```diff
@@ -3,5 +3,12 @@
   "version": "0.4.0-beta.1",
   "description": "WAVS Middleware",
   "author": "Lay3rLabs Team",
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
```

Comment:
please make this a dev dependency

We only need this internally, and @Reecepbcups wants to use this to publish contracts.
I will approve to merge as a dev-dependency, but please check with him for other adjustments